### ISSUE TITLE
.postMessage API (feature request)

### ISSUE THREAD

**Issue by serapath** [postverta/postverta#2]
Projects like http://codesandbox.io and many others can be **embedded**.
I'm in an open source community were we try to teach people from underprivileged backgrounds all over the world how to code. One of the apps we are using is http://wizardamigos.com/workshop_app

In this app, we embed youtube to show lessons.
We embed the gitter chat to support learners
We embed codesandbox to give learners a quick way to practice what they learn.

It would be awesome to embed **postverta**  in these kind of contexts and in a primitive sense this is already possible by displaying at inside an iframe.

The interesting part is - if the rest of the app can understand to some degree what is happening inside the **postverta** iframe and react - and also to some degree control what is happening.

Basically - I imagine, that the **postverta** app would emit events about whats going on and it would listen to some messages to enable to use **postverta** through an **API**

The **API** would be accessible on the **`iframe`** through the `.postMessage` interface

https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage

**Comment by angl** [comment]
This is a very interesting idea. We do have some plan to make Postverta embeddable (based on the oembed standard). I will look into the postMessage interface.

**Comment by angl** [comment]
@serapath We now have an embed API in the works and will release it in a couple of weeks. Meanwhile, would you be willing to give it a spin as a private beta tester? Would love to know what you think about the interfaces and how you might find it useful in your case. If you are interested, please contact me at angl@postverta.com. Thanks!

**Comment by serapath** [comment]
Hey :-) Yes I am interested. Sorry, I only saw your answer now.
Is it documented somewhere? How could I get started? :-)
